proto/storage/auth_provider.proto

```diff
@@ -37,4 +37,9 @@ message AuthProvider {
     repeated RequiredAttribute required_attributes = 11;
 
     Traits traits = 12;
+
+    // Specifies claims from IdP token that will be copied to Rox token attributes.
+    // For example, we can specify nested array claim "realm_access.roles" to be mapped to "groups" attribute.
+    // Currently we only support this feature only for OIDC auth provider.
```

Comment:
This proto file is the only documentation we expose for this feature. Hence I strongly suggest:
* Format it for readability, both in text and in API reference (you can use [this](https://github.com/stackrox/stackrox/blob/a99e49ee0bb165573a04c41af76c7018323af4d2/proto/api/v1/role_service.proto#L201-L228) for inspiration and see its rendering issues [here](https://staging.demo.stackrox.com/main/apidocs#tag/RoleService/operation/RoleService_ComputeEffectiveAccessScope)
* Give examples of what's possible and what's not, what you do in the PR description
* Explicitly say what both `strings` mean, i.e., the first is the dot separated path to the claim in the IdP token and the second is the key name under `Attributes` in the Rox token.